src/adapter.rs

```diff
@@ -221,10 +225,12 @@ impl Adapter {
             self.device.address.clone(),
         ]);
 
+        let narrow_mode =  frame.size().width < 72;
```

Comment:
why `72` exactly ? the help popup and the auth popup will not work for width < 80, maybe we need to change that as well

Reply:
Added to configuration file.